### PR TITLE
Update description of 290/390-p8

### PR DESCRIPTION
**Pull request by ymzhang-neo**
Dear @kevinamstutz , I include a few suggestions for updating Project 8 of 290 and 390. I will include inline comments explaining my suggestions. Thanks!

### CODE REVIEW

File: 08-projects.Rmd [hunk 1]
```diff
@@ -3984,7 +3984,7 @@ DayOfWeek:  0
...
 
-**Context:** This is the first part in a series of projects that are designed to exercise skills around UNIX utilities, with a focus on writing bash scripts and `awk`. You will get the opportunity to manipulate data without leaving the terminal. At first it may seem overwhelming, however, with just a little practice you will be able to accomplish data wrangling tasks really efficiently. 
+**Context:** This is the last part in a series of projects that are designed to exercise skills around UNIX utilities, with a focus on writing bash scripts and `awk`. You will get the opportunity to manipulate data without leaving the terminal. At first it may seem overwhelming, however, with just a little practice you will be able to accomplish data wrangling tasks really efficiently.
```
Comment (ymzhang-neo) [inline 08-projects.Rmd:3987]: I change "first" to "last" as Project 9 seems to be about SQL.

File: 08-projects.Rmd [hunk 2]
```diff
@@ -4010,7 +4010,9 @@ An example of the data for the year 1987 can be found [here](https://www.datadep
 
 ##### 1. Let's say we have a theory that there are more flights on the weekend days (Friday, Saturday, Sunday) than the rest of the days, on average. We can use awk to quickly check it out and see if maybe this looks like something that is true!
 
-##### Write a line of `awk` code that, prints the total number of flights that occur on weekend days, followed by the total number of flights that occur on the weekdays. Complete this calculation for 2008 using the `2008.csv` file.
+##### Write a line of `awk` code that, prints the _total number of flights_ that occur on weekend days, followed by the _total number of flights_ that occur on the weekdays. Complete this calculation for 2008 using the `2008.csv` file.
```
Comment (ymzhang-neo) [inline 08-projects.Rmd:4013]: In the first paragraph of Question 1, there is the phrase "on average". I italicize the two phrases here for clarification.

File: 08-projects.Rmd [hunk 3]
```diff
@@ -4046,9 +4048,9 @@ head airports.csv
 
 **Note:** You may be wondering *why* we are asking you to do this. This sort of situation (where you need to deal with quotes) happens a lot! It's important to practice and learn ways to fix these things.
 
-**Hint:** You could use `gsub` within `awk` to replace '"' with ''.
+**Hint:** You could use `gsub` within `awk` to replace '"' with ''. You can find how to use `gsub` [here](https://www.gnu.org/software/gawk/manual/html_node/String-Functions.html).
```
Comment (ymzhang-neo) [inline 08-projects.Rmd:4051]: There seems no `gsub` explanation in the awk section of the Examples Book. Hence I include a link to the manual.

File: 08-projects.Rmd [hunk 4]
```diff
@@ -4046,9 +4048,9 @@ head airports.csv
...
 
-**Hint:** If you leave out the column number argument to `gsub` it will apply the substitution to every field in every column.
+**Hint:** If you leave out the column number argument to `gsub` it will apply the substitution to every field in every column.
```
Comment (ymzhang-neo) [inline 08-projects.Rmd:4053]: Ah, I did not mean to delete this space! Dear @kevinamstutz , is there a way that I could revert this change? I hope it would not lead to some strange paragraph organization.

Comment (kevinamstutz) [reply 08-projects.Rmd:4053]: The only way I know would be to checkout your pull request branch and remove the extra space and then commit the change :) No need though, an extra space won't hurt a thing!
